rpcs3/Emu/system_progress.hpp

```diff
@@ -15,21 +27,57 @@ extern atomic_t<bool> g_system_progress_stopping;
 class scoped_progress_dialog final
 {
 	// Saved previous value
-	const char* const m_prev;
+	const char* m_prev;
+	u32 m_prev_update;
```

Comment:
Just use a static ID counter. Much easier to understand than some "update counter"

Reply:
Like `m_id = counter++;` in constructor.
And simply set g_progr.last_user_id on update.

Reply:
Actuallly this merhod does not wirk on a large scale with 3 or more users.

Reply:
Actually this method was flawed, and using the previous method with a tweak allows it it to be perfect.